docs/reference/toc.yml

```
@@ -0,0 +1,9 @@
+toc:
+  - file: index.md
+    children:
+        - file: set-up-apm-ios-agent.md
+        - file: supported-technologies.md
+        - file: instrumentation.md
+        - file: configuration.md
+        - title: Troubleshooting
+          crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/ios/index.md
```

I think you need to flatten the nav structure (similar to https://github.com/elastic/elastic-agent/pull/9953#pullrequestreview-3226598623):

```suggestion
toc:
  - file: index.md
  - file: set-up-apm-ios-agent.md
  - file: supported-technologies.md
  - file: instrumentation.md
  - file: configuration.md
  - title: Troubleshooting
    crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/ios/index.md
```